src-ui/src/locale/messages.fr_FR.xlf

```diff
@@ -1537,11 +1537,23 @@
         </context-group>
         <target state="final">Télécharger l'original</target>
       </trans-unit>
+      <trans-unit id="8312409092917397847" datatype="html">
+        <source>Redo OCR</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/components/document-detail/document-detail.component.html</context>
+          <context context-type="linenumber">34</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/components/document-list/bulk-editor/bulk-editor.component.html</context>
+          <context context-type="linenumber">90</context>
+        </context-group>
+        <target state="translated">Refaire l’OCR</target>
```

Comment:
Relancer

Reply:
@llexaxell please submit any suggestions on crowdin, thank you https://crowdin.com/project/paperless-ngx